.github/workflows/libcxx-build-and-test.yaml

```
@@ -38,11 +38,11 @@ env:
   # LLVM POST-BRANCH bump version
   # LLVM POST-BRANCH add compiler test for ToT - 1, e.g. "Clang 17"
   # LLVM RELEASE bump remove compiler ToT - 3, e.g. "Clang 15"
-  LLVM_HEAD_VERSION: "19"   # Used compiler, update POST-BRANCH.
-  LLVM_PREVIOUS_VERSION: "18"
-  LLVM_OLDEST_VERSION: "17"
+  LLVM_HEAD_VERSION: "20"   # Used compiler, update POST-BRANCH.
+  LLVM_PREVIOUS_VERSION: "19"
+  LLVM_OLDEST_VERSION: "18"
   GCC_STABLE_VERSION: "13"
```

```suggestion
  GCC_STABLE_VERSION: "14"
```
Just an off-topic question: Shouldn't this value be 14?

---

Thanks for pointing out! This env variable might be unused. It seems to me that the actual version for gcc is configured [here](https://github.com/llvm/llvm-project/blob/main/libcxx/utils/ci/docker-compose.yml#L4). I will double check my claim.